src/lib.rs

```diff
@@ -503,7 +511,7 @@ impl Future for BackgroundTask {
             match maybe_maybe_item {
                 Some(Some(item)) => self.queues[item.level].push(item),
                 Some(None) => self.quitting = true, // Explicit close.
-                None => self.quitting = true, // The sender was dropped.
+                None => self.quitting = true,       // The sender was dropped.
```

Comment:
Questionable choice. I couldn't find a [rustfmt option](https://rust-lang.github.io/rustfmt/) that disables it though.